### PR TITLE
fix email not showing in create account screen

### PR DESCRIPTION
**Pull request by antsgar**
**Description**

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Jira link:

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [ ] UI components visually reviewed on desktop view
- [ ] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

**Review by Duncan-Brain — APPROVED** [review]
Looks good to me!

**Review by SayakaOno — APPROVED** [review]
Looks good!

**Comment by Duncan-Brain** [comment]
@antsgar this good to merge in?

File: packages/webapp/src/containers/Navigation/index.jsx [hunk 1]
```diff
@@ -18,23 +18,19 @@ import PureNavigation from '../../components/Navigation';
...
+  const historyLocation = useHistoryLocation(history);
+  const isCustomSignupPage = historyLocation.state?.component === CUSTOM_SIGN_UP;
   const isFarmSelected = useIsFarmSelected();
   const { navigation, notification } = useSelector(showedSpotlightSelector);
   const resetSpotlight = () => {
     dispatch(setSpotlightToShown(['notification', 'navigation']));
   };
-  const hideNavBar = [CUSTOM_SIGN_UP, ENTER_PASSWORD_PAGE, CREATE_USER_ACCOUNT].includes(
-    history.location?.state?.component,
-  );
```
Comment (antsgar) [inline packages/webapp/src/containers/Navigation/index.jsx:37]: I'm kinda wondering if we should leave this change in anyway, seems odd that we hide the nav bar only on the first step? @Duncan-Brain @kathyavini @SayakaOno any thoughts?

Comment (Duncan-Brain) [reply packages/webapp/src/containers/Navigation/index.jsx:37]: I like the nav bar on the non login screens personally (even though it does nothing just aesthetics). So I like the revert personally.

**Pull request merged**